src/PKSim.Core/Snapshots/Protocol.cs

```diff
@@ -9,7 +9,7 @@ public class Protocol : ParameterContainerSnapshotBase
 
       public DosingIntervalId DosingInterval { get; set; }
       public string TargetOrgan { get; set; }
-      public string TargetCompartment { get; internal set; }
```

Comment:
Don't ask me why it was there at the first place and why this prevents the entry from being EXPORTED. I would understand no import but export??
Tests do not tests the save and load to file so that's why this was not caught